docs/rules/space-before-keywords.md

```diff
@@ -30,75 +30,75 @@ the keywords `else`, `while` (do...while), `finally` and `catch`. The default va
 This rule will allow keywords to be preceded by an opening curly brace (`{`). If you wish to alter
 this behaviour, consider using the [block-spacing](block-spacing.md) rule.
 
-The following patterns are considered errors when configured `"never"`:
+Examples of **incorrect** code for this rule with the default `"always"` option:
 
 ```js
-/*eslint space-before-keywords: ["error", "never"]*/
+/*eslint space-before-keywords: ["error", "always"]*/
+/*eslint-env es6*/
 
 if (foo) {
     // ...
-} else {}
-
-do {
+}else {}
 
-}
-while (foo)
+const foo = 'bar';let baz = 'qux';
 
-try {} finally {}
+var foo =function bar () {}
 
-try {} catch(e) {}
+function bar() {
+    if (foo) {return; }
+}
 ```
 
-The following patterns are not considered errors when configured `"never"`:
+Examples of **correct** code for this rule with the default `"always"` option:
 
 ```js
-/*eslint space-before-keywords: ["error", "never"]*/
+/*eslint space-before-keywords: ["error", "always"]*/
+/*eslint-env es6*/
 
 if (foo) {
     // ...
-}else {}
+} else {}
 
-do {}while (foo)
+(function() {})()
```

Comment:
Those examples looks incorrect for this rule. Was this meant for some other rule?